Sources/Parsing/Parsers/Prefix.swift

```diff
@@ -44,7 +44,7 @@ where
   public init(
     minLength: Int = 0,
     maxLength: Int? = nil,
-    while predicate: ((Input.Element) -> Bool)? = nil
+    while predicate: @escaping (Input.Element) -> Bool
```

Comment:
This is the one where `where` is required. Figured `Prefix()` doesn't make a lot of sense on its own.